_posts/2022-09-02-tenth-birthday-celebration-recap.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: post
+title: "Trino's tenth birthday celebration recap"
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+---
+
+What an exciting month we had in August! August marked the ten-year birthday of
+the Trino project! Don't worry if you missed all the excitment as we've
+condensed it all in this post!
+
+![](/assets/blog/trino-tenth-birthday/creators.jpeg)
+
+<!--more-->
+
+## Blog posts
+
+We felt it necessary to chronicle the larger events that happened in the last
+decade of the project through the lens of where we are today. Here are few of
+the posts we wrote around this exciting occasion:
+
+* [Why leaving Facebook/Meta was the best thing we could do for the Trino Community]({% post_url 2022-08-02-leaving-facebook-meta-best-for-trino %})
+* [A decade of query engine innovation]({% post_url 2022-08-04-decade-innovation %})
+* [Happy tenth birthday Trino!]({% post_url 2022-08-08-trino-tenth-birthday %})
+
+Some exciting aspects to these blogs were sharing them on HackerNews. The
+Facebook and the query innovation posts both hit the front page. This resulted
+in one of the largest amount of page views on the Trino website in a given day
+(>25k views).
+
+![](/assets/blog/trino-tenth-birthday/hn-top.png)
+![](/assets/blog/trino-tenth-birthday/hn-9.png)
+
+## Trino ten-year timeline video
+
+Another way we celebrated was creating an epic ten-year montage video that
+chronicled the incredible journey starting with the Presto project's humble
```

Comment:
change sentence to present tense.. the video still exists

video that chronicles the incredible journey from the humble beginnings as Presto to the success that Trino is today.